rllib/agents/slateq/slateq_tf_policy.py

```diff
@@ -206,7 +204,8 @@ def build_slateq_stats(policy: Policy, batch) -> Dict[str, TensorType]:
         "next_q_target_slate": policy._next_q_target_slate,
         "next_q_target_max": policy._next_q_target_max,
         "target_clicked": policy._target_clicked,
-        "td_error": policy._mean_td_error,
+        "td_error": policy._td_error,
```

Comment:
if td error is a tensor, how is this supposed to be logged as opposed to mean_td_error which appears to be a scalar?

Reply:
`td_error` needs to have the same batch dim as the row-indices from the buffer, so we know, which rows to update priorities for.

`mean_td_error` is "only" used for reporting.

Reply:
In other words: All off-policy algos with a prio. replay buffer must log this stats from their `stats_fn`.

Reply:
oh gotcha -- That makes sense.